.aspell.en.pws

```diff
@@ -0,0 +1,293 @@
+personal_ws-1.1 en 3
+AAAAB
+ABI
+acpi
+adduser
+adm
+Ansible
+apic
+apiGroups
+apiVersion
+AWS
+aYAAwhn
+bbb
+BhsKcFr
+bqv
+bWxnyp
+cdrom
+cef
+chown
+chpasswd
+cirros
+claimName
+cloudinitdisk
+cloudinitnocloud
+cloudInitNoCloud
+cloudinitvolume
+ClusterIP
+clusternetwork
+ClusterRole
+config
+CPUs
+datasource
+dev
+Dev
+DfPKQZGk
+DNmGruP
+Dockerfile
+dodecacore
+DomainSpec
+dR
+EAAAADAQABAAABAQC
+ee
+ef
+eGHwAWO
+emptyDir
+emptydisk
+emptyDisk
+emptydiskvolume
+ENRjg
+ESo
+explicity
+externalTrafficPolicy
+fc
+fdc
+fEOcxCCt
+filesystem
+FirstSeen
+FIXME
+fx
+GCP
+Gi
+GiB
+GORvegdWZuXgASKVpXeI
+hc
+hpet
+Hy
+hyperv
+img
+init
+init's
+io
+IOThreads
+ip
+iqn
+iscsi
+iSCSI
+JBNZuDzTZSF
+jUg
+KMGrS
+kRIc
+kube
+kubeconfig
+KUBECONFIG
+kubectl
+Kubernetes
```

Comment:
remove

Reply:
This needs to stay I suppose. As it is a correct and recognized word.